src/main/java/eu/greev/dcbot/ticketsystem/interactions/commands/GetTickets.java

```diff
@@ -1,22 +1,37 @@
 package eu.greev.dcbot.ticketsystem.interactions.commands;
 
+import eu.greev.dcbot.ticketsystem.entities.ScrollEntity;
 import eu.greev.dcbot.ticketsystem.entities.Ticket;
 import eu.greev.dcbot.ticketsystem.service.TicketService;
 import eu.greev.dcbot.utils.Config;
 import net.dv8tion.jda.api.EmbedBuilder;
 import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.entities.emoji.Emoji;
 import net.dv8tion.jda.api.events.Event;
 import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.components.buttons.Button;
 
 import java.awt.*;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.TimeUnit;
 
 public class GetTickets extends AbstractCommand {
+    public static final List<ScrollEntity> PAGE_SCROLL_CACHE = new ArrayList<>();
 
     public GetTickets(Config config, TicketService ticketService, EmbedBuilder missingPerm, JDA jda) {
         super(config, ticketService, missingPerm, jda);
-    }
 
+        new Timer().schedule(new TimerTask() {
+            @Override
+            public void run() {
+                PAGE_SCROLL_CACHE.removeIf(entity -> System.currentTimeMillis() - entity.getTimeCreated() > TimeUnit.HOURS.toMillis(12));
```

Comment:
Instead of System.currentTimeMillis and doing some calculations with it use instand.now() and use the before and after methodes